### ISSUE TITLE
[Assistive tech and device support] Pages require sensory characteristics to be understood or operable. (11.11.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54698]
#### VFS team name
GovCIO-VFEP

#### VFS product name
GI Bill Comparison Tool

#### VFS feature name
Institution Ratings System

#### Point of Contact/Reviewers
Brian DeConinck - @briandeconinck - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Pages require sensory characteristics to be understood or operable.

### Issue Details
When viewing an institution details page, in the Veteran Ratings section, for each accordion (Learning Experience, GI Bill Support, Veteran Community, Overall Experience) the stars and numerical rating are not announced by screen readers. This means the summary rating for each category is not provided in any form for screen reader users.

### Link, screenshot or steps to recreate
Looking at the University of Maryland College Park profile, the Learning Experience rating is coded as:

```
<div aria-hidden="true" class="star-heading">
     <div class="vads-u-display--inline-block">
          <i class="fas fa-star vads-u-color--gold-darker vads-u-margin-right--0p2"></i>
          <i class="fas fa-star vads-u-color--gold-darker vads-u-margin-right--0p2"></i>
          <i class="fas fa-star vads-u-color--gold-darker vads-u-margin-right--0p2"></i>
          <i class="far fa-star vads-u-color--gold-darker vads-u-margin-right--0p2"></i>
     </div>
     <span class="vads-u-margin-left--1">2.9</span>
</div>
```

Wrapping the star icons in `aria-hidden` makes sense, but wrapping the numerical rating in `aria-hidden` means that number never gets passed to screen readers.

### VA.gov Experience Standard
[Category Number 11, Issue Number 11](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 1.3.3_A

### Platform Recommendation
Move the numerical rating out of the `aria-hidden`. A few additional comments:
- There might need to be a little additional context provided to screen reader users. Something like "2.9 out of 4" would be sufficient.
- With where the stars/numerical rating is placed in the DOM, I'm not certain it will be announced in an order that makes sense --- it might come before the accordion itself. Definitely test with a screen reader before releasing this fix to verify that everything is announced in a sensible order.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.
